lib/travis/build/script/matlab.rb

```diff
@@ -32,12 +32,13 @@ def setup
           sh.fold 'matlab_install' do
             # Execute helper script to install runtime dependencies
             sh.raw "wget -qO- --retry-connrefused #{MATLAB_DEPS_LOCATION}" \
-                  ' | sudo -E bash -s -- $TRAVIS_MATLAB_VERSION'
+                  ' | travis_retry sudo -E bash -s -- $TRAVIS_MATLAB_VERSION'
 
             # Invoke the ephemeral MATLAB installer that will make a MATLAB available
             # on the system PATH
             sh.raw "wget -qO- --retry-connrefused #{MATLAB_INSTALLER_LOCATION}" \
-                  ' | sudo -E bash -s -- --release $TRAVIS_MATLAB_VERSION'
+                  ' | sudo -E bash -s -- --release $TRAVIS_MATLAB_VERSION' \
+                  ' --error-on-activation-failure'
```

Comment:
I think it might help to check the error code and provide some additional help text. All it does currently is say something pretty generic like:

"An error occurred during installation: activation failed"

You might consider adding some additional text when the exit code == 2, like:

"Currently, MATLAB builds are available only for public projects. If you have any questions or concerns, please contact MathWorks at continuous-integration@mathworks.com."